tests/helix/send-to-helix-inner.proj

```diff
@@ -13,8 +13,7 @@
     <!-- Disable playwright tests on helix/linux. Issue: https://github.com/dotnet/aspire/issues/4623 -->
     <DisablePlaywrightTests Condition="'$(OS)' != 'Windows_NT'">true</DisablePlaywrightTests>
 
-    <GlobalJsonContent>$([System.IO.File]::ReadAllText('$(RepoRoot)global.json'))</GlobalJsonContent>
-    <DotNetCliVersion>$([System.Text.RegularExpressions.Regex]::Match($(GlobalJsonContent), '(%3F&lt;="dotnet": ").*(%3F=")'))</DotNetCliVersion>
+    <DotNetCliVersion>$(DotNetSdkVersionForTesting)</DotNetCliVersion>
```

Comment:
We'll want to make sure now that we update that property so that it at least equals the one in global.json. I wonder if we can add a target somewhere in our build that detects if we've fallen behind and if so to fail the build...

No need to add that in this change though, don't block this PR for that.

Reply:
Once we get the version from a `dotnet/sdk` dependency, it should always(?) be newer than `global.json`.